winrt/lib/text/CanvasFontFace.abi.idl

```diff
@@ -76,12 +76,24 @@ namespace Microsoft.Graphics.Canvas.Text
     [version(VERSION)]
```

Comment:
When we add new APIs, our procedure is to also add documentation. Would you be able to do this? 
Our CONTRIBUTING.md contains some general info about validating changes, and touches on this.
More specifically, to run the doc generation step:
* Run builddocs.cmd from tools\docs. 
* It not succeed (expected) when there are docs missing. Stubs for the missing elements will be written to obj/docs/MissingDocs.xml.
* Copy the stub elements from MissingDocs.xml and paste them in winrt/docsrc/CanvasFontFace.xml alongside the others.
* Fill in the stubs with some documentation text.
* Run builddocs.cmd again; it should succeed.